src/coreComponents/constitutive/capillaryPressure/CapillaryPressureBase.hpp

```diff
@@ -124,7 +124,7 @@ class CapillaryPressureBase : public ConstitutiveBase
   arrayView1d< string const > phaseNames() const { return m_phaseNames; }
 
   arrayView3d< real64 const > phaseCapPressure() const { return m_phaseCapPressure; }
-  arrayView4d< real64 const > dPhaseCapPressure_dPhaseVolFraction() const { return m_dPhaseCapPressure_dPhaseVolFrac; }
+  arrayView4d< real64 const > dPhaseCapPressureDPhaseVolFraction() const { return m_dPhaseCapPressure_dPhaseVolFrac; }
```

Comment:
New derivative style, fine by me though.

Reply:
I have forgotten this correction.
The underscore is back.